CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
 
 ## [Unreleased]
 
+### Added
+
+- EmbeddedArtemisInitializer class
```

Comment:
The description of the change in the change log shoudl describe what the change is (and, probably, why), not just what classes have been added.